Singularity/Singularity/Platform/ProduceResource.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using Singularity.Map;
+using Singularity.Resources;
+using Singularity.Units;
+
+namespace Singularity.Platform
+{
+
+    public class ProduceWellResource : APlatformAction
+    {
+
+        // The ResourceMap is needed for actually 'producing' the resources.
+        private ResourceMap mResourceMap;
+
+        public ProduceWellResource(PlatformBlank platform, ResourceMap resourceMap) : base(platform)
+        {
+            mResourceMap = resourceMap;
+        }
+
+        public override List<JobType> UnitsRequired { get; } = new List<JobType>{ JobType.Production };
```

Comment:
Why is this named UnitsRequired if you return a list of jobs? Whats the intention behind that?

Reply:
it's the list of what type of worker-units is required at this platform. In this case it's a producing platform, so obviously it's producing units, for defending platforms it's Defense-type units (the kinetic one's at least)